content/notebooks/hydrology/04_Time_series_analysis.ipynb

```diff
@@ -0,0 +1,1944 @@
+{
```

Comment:
<p>on PAVICS beta image this is giving a blank figure </p><p>out is all nan  </p>

<pre class="ql-syntax" spellcheck="false">`array([nan, nan, nan, nan, nan, nan])
</pre><p>` </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/PAVICS-landing/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>I think xclim changed and now the missing checks are applied. We can get the old results back by adding <code>with xclim.set_options(check_missing="skip"):</code></p>